games/openrct2/Portfile

```diff
@@ -4,10 +4,10 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        OpenRCT2 OpenRCT2 0.3.5 v
-github.tarball_from archive
+github.setup        OpenRCT2 OpenRCT2 0.3.5.1 v
+github.master_sites https://codeload.github.com/${github.author}/${github.project}/zip/refs/tags/${git.branch}?dummy=
```

Comment:
I would find it interesting if `[string range ${extract.suffix} 1 end]` (remove dot) was taken into account in `github.tarball_from`. Otherwise, need a better way to set up extraction when there's multiple formats.

Reply:
`extract.suffix .zip` should be removed while you're at it. `use_zip yes` sets that for you.

Reply:
Should be good soon.

Reply:
With the workaround removed, it's downloading the tarball from `https://codeload.github.com/OpenRCT2/OpenRCT2/legacy.tar.gz/v0.3.5.1?dummy=`.

Reply:
That should only be the case if you did not specify `github.tarball_from archive`.